### ISSUE TITLE
[Bug]: Wrong behavior for using file paths in remote "run configuration" (hop server/docker long lived).

### ISSUE THREAD

**Issue by transformandodados** [apache/hop#4161]
### Apache Hop version?

2.9.0

### Java version?

openjdk 11.0.23

### Operating system

Linux

### What happened?

I'm having incorrect behavior for "pipeline and workflow" executed on a "Hop Server" or "Hop Server Docker (long lived)" when I use files (csv/txt/excel). In the example below, I'm creating a "pipeline" that writes a txt file to the ${PROJECT_HOME} directory. When I run it locally, I get the expected result: a txt file named "filename_" created inside the ${PROJECT_HOME} folder.

Now, when I run it on a "Hop server", the file is being created inside the root folder of the home, which in the case of Docker is "/opt/hop/" in a strange subdirectory: ${DATA_PATH_1}, as shown in the images below:
 
![image](https://github.com/user-attachments/assets/8578fe16-1f37-4f27-b8bd-0b026d2935f4)

These are the settings of my “Pipeline Run Configuration” setting the “Named resources” “source” and “target” as read in the documentation: 
![image](https://github.com/user-attachments/assets/753c2177-4ad3-4cfd-9c15-eb489d558f98)

https://hop.apache.org/manual/latest/pipeline/pipeline-run-configurations/native-remote-pipeline-engine.html

The expected thing was that the file would be saved in “/files/” inside my “Hop Server” container. 
 
![image](https://github.com/user-attachments/assets/18214e35-f9b7-412e-8bf6-fe9064657976)

![image](https://github.com/user-attachments/assets/59415438-1f5c-457f-8e4d-3d44eab1e7e5)

But it was not in /files but in “/opt/hop/${DATA_PATH_1}”
![image](https://github.com/user-attachments/assets/8175e355-02cd-47e1-a9a3-69f38597536f)

This is definitely not the expected behavior, we want to use Hop in production, but this is causing an issue.
We would like some help to resolve this.
Thank you very much!

### Issue Priority

Priority: 1

### Issue Component

Component: Hop Server

**Comment by hansva** [comment]
I would suggest using a variable different from ${PROJECT_HOME} to reference data files.
For example the ${files} variable you created, you can then use the remote run configuration to overwrite the value to the correct path on the remote server. This should get you unblocked for now.

**Comment by transformandodados** [comment]
Thank you for your response!
Any information I put in the variables it always creates within the root of the hop binaries, /opt/hop/${DATA_PATH_1}

**Comment by hansva** [comment]
Even if you use a different variable inside the text file output?

**Comment by transformandodados** [comment]
Exactly. Even if you put a path in the file without a variable. Putting a variable. Setting it or not or "named resources, source/target".
In any scenario I have the same result, which is in the hop realm: opt/hop/${DATA_PATH_1} --> yes, it creates a folder with that name.
The only scenario with a different result, which I understand is not correct for several reasons, is to manually set the value of the ${DATA_PATH_1} variable to a valid path, then it works.

**Comment by CuproAT** [comment]
today we hit the same issue with 2.9

**Comment by pramodgsable** [comment]
Same here. We are setting the target filepath by manually assigning values to ${DATA_PATH_1} variable

**Comment by hansva** [comment]
duplicate of https://github.com/apache/hop/issues/7209
